mindmeld/components/dialogue.py

```diff
@@ -717,12 +718,13 @@ class AutoEntityFilling:
     _logger = mod_logger.getChild("AutoEntityFilling")
     """Class logger."""
 
-    def __init__(self, entrance_handler, form, app):
+    def __init__(self, handler, form, app):
```

Comment:
We should update AutoEntityFiling doc (line 714)

Reply:
Even if all attributes are private?

Reply:
I see, maybe update to remove attribute "app" and just create a docstring for ``__init__`` instead.